src/Spain/Mews.Fiscalizations.Spain/Model/Response/ErrorResult.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using FuncSharp;
+using Mews.Fiscalizations.Spain.Model.Enums;
```

Comment:
removed unused using.

Reply:
Strange it shoould be a folder inside the same project...Oh I just didn't save the file before commit. my bad